### PR TITLE
fixed some typos

### PR DESCRIPTION
**Pull request by zhigunenko-dsr**
Signed-off-by: Nikita Zhigunenko <nikita.zhigunenko@dsr-corporation.com>

### CODE REVIEW

**Comment by hyperledger-github** [comment]
Could one of the committers please verify this patch?

**Comment by ashcherbakov** [comment]
test this please

File: indy_node/server/node.py [hunk 1]
```diff
@@ -333,7 +333,7 @@ def executeDomainTxns(self, ppTime, reqs: List[Request], stateRoot,
         :param ppTime: the time at which PRE-PREPARE was sent
         :param req: the client REQUEST
         """
-        return self.default_executer(DOMAIN_LEDGER_ID, ppTime, reqs,
+        return self.default_executor(DOMAIN_LEDGER_ID, ppTime, reqs,
```
Comment (ashcherbakov) [inline indy_node/server/node.py]: This is an API method defined in Plenum...
Please be careful when renaming API methods

File: indy_node/server/pool_manager.py [hunk 2]
```diff
@@ -9,7 +9,7 @@ class HasPoolManager(PHasPoolManager):
     def __init__(self, ha=None, cliname=None, cliha=None):
         self.poolManager = TxnPoolManager(self, ha=ha, cliname=cliname,
                                           cliha=cliha)
-        self.requestExecuter[POOL_LEDGER_ID] = \
+        self.requestExecutor[POOL_LEDGER_ID] = \
```
Comment (ashcherbakov) [inline indy_node/server/pool_manager.py]: This is an API method defined in Plenum...

**Comment by andkononykhin** [comment]
test this please

**Comment by andkononykhin** [comment]
test this please

**Comment by andkononykhin** [comment]
test that please

**Comment by brentzundel** [comment]
Upon reviewing, I found this PR acceptable, but now that it is so out of date, and has so many conflicts, perhaps it would be best to just close it.

**Comment by brentzundel** [comment]
Closing PR

**Comment by brentzundel** [comment]
This PR was very out of date and had many conflists.

**Pull request closed without merging**